setup.py

```diff
@@ -13,7 +13,7 @@
 
 
 install_requires = (
-    'Django>=1.11,<3.1',
+    'Django>=1.11,<3.2',
     'richenum',
     'six',
```

Comment:
```suggestion
```
Also of note, we want to remove the ```six``` references. This resulted in some very minor changes to remove ```six.iteritems()``` and ```six.string_types``` / ```six.integer_types```.